File: EJERCICIO_2/EJERCICIOSPHP/plan.php
<?php

class Plan {
    private $type;
    private $price;
    private $services;

    public function __construct($type, $price, $services) {
        $this->type = $type;
        $this->price = $price;
        $this->services = $services;

    }
    

    public function getType() {
        return $this->type;
    }

    public function getPrice(){
        return $this->price;
    }
    public function getServices(){
        return $this->services;
    }
}

    ?>

File: EJERCICIO_2/EJERCICIOSJAVA/EJERCICIOJAVA/Plan.java
package EJERCICIOJAVA;
public class Plan {
    private String type;
    private double price;
    private String services;

    public Plan(String type, double price, String services) {
        this.type = type;
        this.price = price;
        this.services = services;
    }

    public String getType() {
        return type;
    }

    public double getPrice() {
        return price;
    }

    public String getServices() {
        return services;
    }
}